scripts/pocs-shell.py

```diff
@@ -190,7 +190,24 @@ def do_reset_pocs(self, *arg):
 
         Does NOT park the mount, nor execute power_down.
         """
-        self.pocs = None
+        if self.pocs is None:
+            print_info("POCS does not have an instance. Create one using 'setup_pocs'.")
+            return
+        elif self.pocs.observatory.mount.is_parked is True:
+            self.pocs = None
+        elif self.pocs.observatory.mount.is_parked is False:
+            print_warning("ATTENTION: The mount is not in the parked position and reset_pocs will not move the mount to park position!")
+            while True:
+                response = input("The unit could be damaged if it is not parked before sunrise. Execute reset_pocs anyway[y/n]?")
+                if response.lower().startswith('y'):
+                    self.pocs = None
+                    print_info("POCS instance discarded. Remember to park the mount later.")
+                    break
+                elif response.lower().startswith('n'):
+                    print_info("Keeping POCS instance.")
+                    break
+                else:
+                    print("A 'yes' or 'no' response is required")
```

Comment:
POCS (mostly the underlying `Observatory`) can support a mode where you don't actually have a mount, which is a bit odd but could be used for things like testing cameras. I could see wanting to reset pocs (especially) in one of those scenarios. So this might be a bit better as:

```python
if self.pocs is None:
    print_info("POCS does not have an instance. Create one using 'setup_pocs'.")
    return
elif self.pocs.observatory.mount and not self.pocs.observatory.mount.is_parked:
    print_warning("ATTENTION: The mount is not in the parked position and reset_pocs will not move the mount to park position!")
    while True:
        response = input("The unit could be damaged if it is not parked before sunrise. Execute reset_pocs anyway [y/n]?")
        if response.lower().startswith('y'):
            print_info("POCS instance discarded. Remember to park the mount later.")
            break
        elif response.lower().startswith('n'):
            print_info("Keeping POCS instance.")
            return
        else:
            print("A 'yes' or 'no' response is required")

# Reset in all other scenarios
self.pocs = None
```